src/cluster/placement/staged_placement.go

```diff
@@ -34,38 +33,36 @@ import (
 var (
 	errNoApplicablePlacement       = errors.New("no applicable placement found")
 	errActiveStagedPlacementClosed = errors.New("active staged placement is closed")
+	errPlacementInvalidType        = errors.New("type assertion failed, corrupt placement")
```

Comment:
nit: type assertion is an implementation detail, this can probably just be "corrupt placement"